lunatrace/bsl/ml/python/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.8
+
+RUN pip install pipenv
+
+ENV PROJECT_DIR /usr/local/src/webapp
+
+WORKDIR ${PROJECT_DIR}
+
+COPY Pipfile Pipfile.lock ${PROJECT_DIR}/
+
+RUN pipenv install --system --deploy
```

Comment:
yeah this is nice, i guess i will stop bothering trying to figure out how to properly build a python artifact. This will probably keep working fine because of the lockfile.